pipelines/exposureQualityCore.yaml

```diff
@@ -0,0 +1,13 @@
+description: |
+  Tier 1 plots and metrics to assess exposure quality
+tasks:
+  analyzeAmpOffsetTaskCore:
+    class: lsst.analysis.tools.tasks.IsrAmpOffsetMetadataAnalysisTask
+    config:
+      connections.outputName: isrAmpOffsetMetadata
+      atools.isrAmpOffsetMetadataMetric: MetadataMetricTool
+      atools.isrAmpOffsetMetadataMetric.metrics:
+        # Format is "metric name in the metadata": units
+        AMPOFFSET_PEDESTALS: adu
```

Comment:
Is there any merit in storing amp-offsets in nanoJanskys, rather than ADUs?

Reply:
Hmm, do we care about cross-instrument comparisons?

Reply:
Maybe? To put this another way, if I did want to convert these per-amp pedestals to nJy after-the-fact, how easy would that be for an end-user?

Reply:
It'd take some effort - not super easy, but I double-checked with @TallJimbo, and he believes `ADU` is fine because people would be mostly interested in it within the instrument.